rllib/policy/policy.py

```diff
@@ -236,6 +267,37 @@ def set_weights(self, weights):
         """
         raise NotImplementedError
 
+    @DeveloperAPI
+    def get_exploration_state(self):
+        """
+        Returns the current exploration state of this policy, which depends
+        on the policy's Exploration object.
+
+        Returns:
+            any: Serializable copy or view of the current exploration state.
+        """
+        if self.exploration is not None:
+            raise NotImplementedError
+
+    @DeveloperAPI
+    def set_exploration_state(self, exploration_state):
+        """Sets the current exploration state of this Policy.
+
+        Arguments:
+            exploration_state (any): Serializable copy or view of the new
+                exploration state.
+        """
+        if self.exploration is not None:
+            raise NotImplementedError
+
+    @DeveloperAPI
+    def is_recurrent(self):
+        """
+        Returns:
+            int: The number of RNN hidden states kept by this Policy's Model.
```

Comment:
Shouldn't this be a bool? Seems to duplicate num_state_tensors below.

Reply:
It's not written/added by me. I think I moved it there (from further down). I'll check, whether it can be removed entirely. ...

Reply:
Ah, you meant the comment. Fixed it.